src/main/java/online/carsharing/service/impl/NotificationServiceImpl.java

```diff
@@ -0,0 +1,127 @@
+package online.carsharing.service.impl;
+
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import online.carsharing.entity.Car;
+import online.carsharing.entity.UserChatId;
+import online.carsharing.repository.car.CarRepository;
+import online.carsharing.repository.user.UserChatIdRepository;
+import online.carsharing.service.NotificationService;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.scheduling.annotation.Async;
+import org.springframework.stereotype.Service;
+import org.telegram.telegrambots.bots.TelegramLongPollingBot;
+import org.telegram.telegrambots.meta.api.methods.send.SendMessage;
+import org.telegram.telegrambots.meta.api.objects.Update;
+import org.telegram.telegrambots.meta.exceptions.TelegramApiException;
+
+@Service
+@RequiredArgsConstructor
+public class NotificationServiceImpl extends TelegramLongPollingBot implements NotificationService {
+
+    private final UserChatIdRepository userChatIdRepository;
+    private final CarRepository carRepository;
+
+    @Value("${bot.botname}")
+    private String botName;
+
+    @Value("${bot.bottoken}")
+    private String botToken;
+
+    @Override
+    public String getBotUsername() {
+        return this.botName;
+    }
+
+    @Override
+    public String getBotToken() {
+        return this.botToken;
+    }
+
+    @Async
+    @Override
+    public void onUpdateReceived(Update update) {
+        if (update.hasMessage() && update.getMessage().hasText()) {
+            String command = update.getMessage().getText().trim();
+
+            if (command.startsWith("/addMe")) {
+                addUserChatId(update);
+            } else if (command.startsWith("/getModel")) {
+                String[] parts = command.split(" ");
+                if (parts.length == 2) {
+                    try {
+                        Long id = Long.parseLong(parts[1]);

```

Comment:
magic numbers

Reply:
I will change it but right now this functional moved to other branch